pkg/http/api.go

```diff
@@ -722,14 +741,12 @@ func (a *api) onGetWorkflowHandler() fasthttp.RequestHandler {
 		})
 }
 
-type workflowActivityHandler = func(ctx context.Context, in *runtimev1pb.WorkflowActivityRequest) (*runtimev1pb.WorkflowActivityResponse, error)
-
-// This is a common handler for TerminateWorkflow, PauseWorkflow and ResumeWorkflow
-func (a *api) onWorkflowActivityHandler(handler workflowActivityHandler) fasthttp.RequestHandler {
```

Comment:
Why was this undone?

Reply:
Yes, this was removed intentionally. I wanted to make the terminate, pause, and resume APIs more future-proof by giving them their own distinct contract and implementation, which means we don't use a common API handler for all of them.